### ISSUE TITLE
Learning an Optimally Reduced Formulation of OPF through Meta-optimization. (arXiv:1911.06784v3 [eess.SP] UPDATED)

### ISSUE THREAD

**Issue by CoffeeKumazaki** [CoffeeKumazaki/arXiv#2198]
Learning an Optimally Reduced Formulation of OPF through Meta-optimization. (arXiv:1911.06784v3 [eess.SP] UPDATED)<br>
https://ift.tt/37FcRSd<br>
<p>With increasing share of renewables in power generation mix, system operators would need to run Optimal Power Flow (OPF) problems closer to real-time to better manage uncertainty. Given that OPF is an expensive optimization problem to solve, shifting computational effort away from real-time to offline training by machine learning techniques has become an intense research area. In this paper, we introduce a method for solving OPF problems, which can substantially reduce solve times of the two-step hybrid techniques that comprise of a neural network with a subsequent OPF step guaranteeing optimal solutions. A neural network that predicts the binding status of constraints of the system is used to generate an initial reduced OPF problem, defined by removing the predicted non-binding constraints. This reduced model is then extended in an iterative manner until guaranteeing an optimal solution to the full OPF problem. The classifier is trained using a meta-loss objective, defined by the total computational cost of solving the reduced OPF problems constructed during the iterative procedure. Using a wide range of DC- and AC-OPF problems, we demonstrate that optimizing this meta-loss objective results in a classifier that significantly outperforms conventional loss functions used to train neural network classifiers. We also provide an extensive analysis of the investigated grids as well as an empirical limit of performance of machine learning techniques providing optimal OPF solutions.</p>
<br>
<br>
via eess.SY updates on arXiv.org http://arxiv.org/<br>
